### ISSUE TITLE
Problem with adanet_objective.ipynb

### ISSUE THREAD

**Issue by ngussek** [tensorflow/adanet#48]
Hi, 

I wanted to run the adanet_objective.ipynb Notebook with Tensorflow 1.12. 
This part throws an error:
`def ensemble_architecture(result):

architecture = result["architecture/adanet/ensembles"]

-->  summary_proto = tf.summary.Summary.FromString(architecture) <-- Error here.

return summary_proto.value[0].tensor.string_val[0]`
There doesnt seem to be a "FromString" method. I found it in very old API documentations, but I couldn't find a replacement in the newer versions. Can somebody please tell me how to replace this line?

Thanks!

**Comment by cweill** [comment]
`tf.Summary` is a protobuf ([see tf.Summary proto file](https://github.com/tensorflow/tensorflow/blob/r1.12/tensorflow/core/framework/summary.proto)), and all protos have a generated Python object with a `.FromString(...)` method. You may need to update your `protobuf` dependency with pip.

**Comment by ngussek** [comment]
Hey, 
thanks for your reply! I checked my protobuf version and everything else I could find somehow... Problem was/is somewhere inside Visual Studio Code or in my config. Pylint shows me the following message: "Class 'Summary' has no 'FromString' member". I never even tried running it because of that error. Now I decided to just launch the script despite the error and yeah ...it  just worked without a problem. 
Sorry, my bad!
